app/ide-desktop/lib/dashboard/tailwind.config.js

```diff
@@ -19,11 +19,14 @@ export default /** @satisfies {import('tailwindcss').Config} */ ({
         /** The default color of all text. */
         // This should be named "regular".
         primary: 'rgb(0 0 0 / 60%)',
+        accent: '#499f4b',
         'hover-bg': 'rgb(0 0 0 / 10%)',
         frame: 'var(--frame-color)',
         'selected-frame': 'var(--selected-frame-color)',
         'ide-bg': '#ebeef1',
         selected: 'rgb(255 255 255 / 40%)',
+        'not-selected': 'rgb(0 0 0 / 15%)',
+        'window-close': '#ff605c',
```

Comment:
use generic names instead of specific names

Reply:
this is the generic name, this is the macos close color